### DISCUSSION TITLE
LDAP issues creating new user

### DISCUSSION THREAD

**Discussion by fabio-pif** [kimai/kimai#4444] [General]
I've installed sfew weeks ago the LDAP extension as described in 
https://www.kimai.org/documentation/ldap.html

and it was working well, i was able to create and login with an AD user.
Now i still hable to login with this user, but when i try with new one i got error message.

To try to debug it, following the instruction i should 
1. switch to APP_ENV=dev mode 
2.  run composer install.

Questions:
- To switch to APP_ENV=dev mode , i've to edit the  .env  file or there is a command line?
- after that, i need simply the run the command "composer install" in the shell console ?
- where i will then find log / debug files?

Thank for any help

Fabio

p.s.:
I'm running version 2.3.0 in PROD mode, on a Ubunto 22.04 server

**Comment by kevinpapst** [comment]
There seem to be some some issues with LDAP, which I am not able to reproduce locally.

I am trying to fix these problems, but need help with testing. 
If you can, give this a try: https://github.com/kimai/kimai/pull/4445

**Comment by fabio-pif** [comment]
Solved with version 2.4.0 and reinstalling LDAP!
Thx
